news/parse/epochtimes.py

```diff
@@ -299,6 +331,236 @@
+    (
+        re.compile(r'http://tuidang.epochtimes.com/index/showpost/id/(\d)*'),
+        '',
+    ),
+    # This observation is made with `url_pattern = 19-12-23-11741126,
+    # 19-12-5-11702000`.
+    (
+        re.compile(r'\s註[:\s\[\d\]]*(?:李洪志師父著作:《轉法輪》|這高度表示安全、' + r'牢不可破。)'),
+        '',
+    ),
+    # This observation is made with `url_pattern = 19-12-21-11736511
+    # 19-12-14-11722295`.
+    (
+        re.compile(r'(?:那?好|最後),歡迎您訂閱和分享我們的頻道.*'),
+        '',
+    ),
+    # This observation is made with `url_pattern = 19-12-20-11735692
+    # 19-12-13-11721610`.
+    (
+        re.compile(r'本文僅代表作者(個人)?觀點。$'),
+        '',
+    ),
+    # This observation is made with `url_pattern = 19-12-19-11733577`.
+    (
+        re.compile(r'嘉賓:[\s\w]*,[\s\w]*主持人:.*$'),
+        '',
+    ),
+    # This observation is made with `url_pattern = 19-12-19-11733207,
+    # 19-12-20-11734006, 19-12-18-11730384, 19-12-18-11731126,
+    # 19-10-16-11592972, 19-5-13-11255964, 19-5-11-11250223,
+    # 20-1-1-11761463, 19-12-30-11756323, 19-12-27-11749380
+    # 19-12-27-11748178`.
+    (
+        re.compile(
+            r'(?:文章來源|大紀元新聞網|主辦單位|作者簡介|醫師小檔案|明慧學校夏'
+            + r'令營時間|直播日期|有問題請聯繫|參考書目|【漣漪小語】|新唐人電視台官網'
+            + r'):.*'
+        ),
+        '',
+    ),
+    # This observation is made with `url_pattern = 19-12-17-11727211`.
+    (
+        re.compile(r'\(\*\*\*插入片段\).*'),
+        '',
+    ),
+    # This observation is made with `url_pattern = 19-12-15-11724589,
+    # 19-12-14-11722684, 19-5-7-11240051, 19-5-11-11250223, 19-12-17-11727172
+    # 19-12-20-11734996, 19-12-27-11748709, 19-8-14-11453112,
+    # 19-12-1-11692633, 19-11-29-11688080, 19-11-13-11653910,
+    # 19-11-13-11653876, 19-10-30-11623041, 19-10-28-11617434,
+    # 19-10-17-11595022`.
+    (
+        re.compile(
+            r'\((?:澳洲墨爾本記者站採訪報導|本文歡迎轉載引用,註明出處'
+            + r'即可|簽[名字]|化名|轉改自美國之音|(?:未|本[章卷])完,(全文)?待續|網站專文|'
+            + r'特別銘謝:.*|《.*敬請關注.*|敬請繼續關注.*|全文完|接[下上]文|前言)\)'
+        ),
+        '',
+    ),
+    # This observation is made with `url_pattern = 19-12-20-11735692,
+    # 19-12-15-11723411.`
+    (
+        re.compile(r'(接上文:)?【(十字路口|美國思想領袖)\s?】[\w\d\s]*\([上中]\),?'),
+        '',
+    ),
+    # This observation is made with `url_pattern = 19-12-15-11723411'`.
+    (
+        re.compile(r'以下視頻是採訪的第[二三]部分\(中文字幕\):(?:中共就是個黑幫組織|面對殘暴年輕人毫不退縮)。'),
+        '',
+    ),
+    # This observation is made with `url_pattern = 19-12-14-11722898`.
+    (
+        re.compile(r',您去大紀元、新唐人網站,就可以看到活動的具體訊息。好的非常感謝各位的收看,我們下次節目再見。(\s?)'),
+        r'\1',
+    ),
+    # This observation is made with `url_pattern = 19-11-4-11631494,
+    # 19-5-3-11232059, 19-12-30-11754661`.
+    (
+        re.compile(r'(?:法輪大法世界各地免費煉功點|觀看|吳熾昌《客窗閒話》)$'),
+        '',
+    ),
+    # This observation is made with `url_pattern = 19-12-20-11734499,
+    # 19-10-23-11606025, 19-10-21-11602441`.
+    (
+        re.compile(r'(?:\s|時間:.*)(?:地址|地點|電話):.*(?:電話|地址|郵箱):.*'),
```

Comment:
Use `\S+` instead of `.*`

Reply:
Because of case 19_10_23_11606025, I couldn't let the second become \S+